.vscode/tasks.json

```diff
@@ -50,6 +69,18 @@
                 "reveal": "silent"
             },
             "problemMatcher": "$msCompile"
+        },
+        {
+            "label": "start research",
+            "type": "shell",
+            "group": "build",
+            "isBackground": true,
+            "command" : "${workspaceFolder}/.vscode/launch_research.sh",
+            "args" : [
+                "${workspaceFolder}/Launcher/bin/Debug",
+                "${workspaceFolder}/Data"
+            ],
+            "problemMatcher": "$msCompile"
```

Comment:
Which should probably add a dependency on the build task for this one? will it open the jupyter notebook viewer automatically?

Reply:
After we run this command, a pop-up window opens and we can open the notebook on a web browser.